packages/flutter_tools/test/commands.shard/hermetic/pub_get_test.dart

```diff
@@ -112,6 +113,45 @@ void main() {
     ProcessManager: () => FakeProcessManager.any(),
     FileSystem: () => fileSystem,
   });
+
+  testUsingContext('pub get triggers localizations generation when generate: true', () async {
+    final File pubspecFile = fileSystem.currentDirectory.childFile('pubspec.yaml')
+      ..createSync();
+    pubspecFile.writeAsStringSync('''
```

Comment:
nit on the identation for this and next `writeAsStringSync`
```
pubspecFile.writeAsStringSync(
  '''
  flutter:
    generate: true
  '''
);
```

Reply:
Fixed, thanks!